firstvoices/backend/permissions/predicates/view.py

```diff
@@ -25,3 +25,7 @@
     ),
     name="has_visible_site",
 )
+
+can_view_hidden_site = Predicate(
+    base.is_at_least_staff_admin | base.has_team_access, name="can_view_hidden_site"
+)
```

Comment:
This is so readable, I love it.

Reply:
I think once you use this on the model permissions, you will need a matching permission filter defined for it as well